msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionForumBean.java

```diff
@@ -38,6 +38,9 @@
 import org.sakaiproject.api.app.messageforums.ui.DiscussionForumManager;
 import org.sakaiproject.api.app.messageforums.ui.UIPermissionsManager;
 
+import org.sakaiproject.time.cover.TimeService;
```

Comment:
I agree with you, we should avoid the use of covers, but to be honest MSGCNTR uses the covers widely in several classes, i think this concern is outside the scope of this patch.